dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -473,6 +481,43 @@ public Response getVersion(@PathParam("id") String datasetId, @PathParam("versio
         }));
     }
 
+    @GET
+    @Path("{id}/uningest")
+    public Response listUningestableFiles(@PathParam("id") String datasetId) {
+        return allowCors(response(req -> ok(uningestInfoService.listUningestableFiles(findDatasetOrDie(datasetId)).stream()
```

Comment:
I don't think that we need cors for this endpoint. Do we want to allow for some third party websites to execute it?